mmgen/models/architectures/sngan_proj/generator_discriminator.py

```diff
@@ -2,100 +2,86 @@
 
 import torch
 import torch.nn as nn
-from mmcv.cnn import ConvModule, xavier_init
-from mmcv.cnn.bricks.activation import build_activation_layer
+from mmcv.cnn import ConvModule, build_activation_layer, xavier_init
+from mmcv.runner import load_checkpoint
+from mmcv.utils import build_from_cfg
 from torch.nn.utils import spectral_norm
 
 from mmgen.models.builder import MODULES
-from mmgen.utils.check_dist_init import check_dist_init
+from mmgen.utils import check_dist_init
+from mmgen.utils.logger import get_root_logger
 from ..common import get_module_device
-from .modules import (SNGAN_DisHeadResBlock, SNGAN_DisResBlock,
-                      SNGAN_GenResBlock)
 
 
 @MODULES.register_module()
 class SNGANGenerator(nn.Module):
 
-    # TODO: why wganpg use str as key?
+    # TODO: add doc here
     _defualt_channels_cfg = {
         32: [1, 1, 1],
         64: [16, 8, 4, 2],
         128: [16, 16, 8, 8, 4, 2]
     }
 
-    # TODO: move this to _base_?
-    _default_res_block_cfg = dict(
-        upsample=True,
-        use_cbn=True,
-        use_norm_para=False,
-        act_cfg=dict(type='ReLU'),
-        norm_cfg=dict(type='BN'),
-        upsample_cfg=dict(type='nearest', scale_factor=2),
-        conv_cfg=dict(
-            kernel_size=3,
-            stride=1,
-            padding=1,
-            act_cfg=None,
-        ),
-        shortcut_cfg=dict(kernel_size=1, stride=1, padding=0, act_cfg=None))
+    _default_blocks_cfg = dict(type='SNGANGenResBlock')
```

Comment:
If the default config is simple, we can just remove it to the `__init___` function.